doc/user/content/sql/create-cluster.md

```diff
@@ -35,163 +35,106 @@ you may not create a cluster with a source and an index.
 We plan to remove this restriction in a future version of Materialize.
 {{< /warning >}}
 
-Importantly, clusters are strictly a logical component; they rely on [cluster
-replicas](/get-started/key-concepts#cluster-replicas) to run dataflows. Said a
-slightly different way, a cluster with no replicas does no computation. For
-example, if you create an index on a cluster with no replicas, you cannot select
-from that index because there is no physical representation of the index to read
-from.
-
-Though clusters only represent the logic of which objects you want to bundle
-together, this impacts the performance characteristics once you provision
-cluster replicas. Each object in a cluster gets instantiated on every replica,
-meaning that on a given physical replica, objects in the cluster are in
-contention for the same physical resources. To achieve the performance you need,
-this might require setting up more than one cluster.
-
-{{< warning >}}
-Clusters containing sources and sinks can have at most one replica.
-
-We plan to remove this restriction in a future version of Materialize.
-{{< /warning >}}
-
-## Managed and unmanaged clusters
-
-A managed cluster is one with a declared size and replication factor, where
+Each cluster has a declared size and replication factor, where
 Materialize is responsible for ensuring the replica set matches the declared
 size and replication factor. The replicas of a managed cluster are visible in
 the system catalog, but cannot be directly modified by users.
 
-An unmanaged cluster requires you to manage replicas manually, by creating and
-dropping replicas to achieve the desired replication factor and replica size.
-The replicas of unmanaged clusters appear in the system catalog and can be
-modified by users.
-
-
-{{< warning >}}
-Managed clusters with sources and sinks only support a replication factor of zero or one.
-
-We plan to remove this restriction in a future version of Materialize.
-{{< /warning >}}
+A clusters size controls the amount of memory, CPU, and disk allocated to each of the clusters replicas.
+The replication factor controls the number of replicas allocated to a cluster.
 
-### Managed clusters
+TALK MORE ABOUT SIZE VS REPLICATION FACTOR HERE
```

Comment:
We have text about this on the `CREATE CLUSTER REPLICA` page that we could move here with some minor adaptations:

```
Materialize is an active-replication-based system, which means you expect each
cluster replica to have the same working set.
With this in mind, when building your Materialize deployment, you can change its
performance characteristics by...
Action | Outcome
---------|---------
 Increase all replicas' sizes | Ability to maintain more dataflows or more complex dataflows
 Add replicas to a cluster | Greater tolerance to replica failure
```